contracts/dynamic-callee-contract/src/contract.rs

```diff
@@ -53,6 +53,11 @@ fn pong_env() -> Env {
     GlobalApi::env()
 }
 
+#[callable_point]
+fn callee_panic() {
```

Comment:
function names should start with a verb. and, this function can be called in this contract if it is not callee. So `callee` is unsuitable. how about `do_panic`, `raise_panic`, `issue_panic`, or something like these?